packages/generator/tests/test_assets/example_renders/measurement_plugin_client/non_streaming_data_measurement_client.py

```diff
@@ -420,11 +442,11 @@ def stream_measure(
         float_in: float = 0.05999999865889549,
         double_array_in: List[float] = [0.1, 0.2, 0.3],
         bool_in: bool = False,
-        string_in: str = "sample string",
+        string_in: str = b"sample string",
```

Comment:
None of these test cases use escapable characters. Please add test cases that use backslash, single quote, and double quote.